src/Objects/MonitorManager.vala

```diff
@@ -20,13 +20,13 @@
  */
 
 public class Display.MonitorManager : GLib.Object {
-    public Gee.LinkedList<Display.VirtualMonitor> virtual_monitors { get; construct; }
-    public Gee.LinkedList<Display.Monitor> monitors { get; construct; }
-
+    public Gee.LinkedList<Display.VirtualMonitor> virtual_monitors { get; set; }
+    public Gee.LinkedList<Display.Monitor> monitors { get; set; }
```

Comment:
Why can these not be `construct` properties?  You can still add to the lists after constructing. You do not need to set new list objects though.

Reply:
Ah, that might have been a leftover from a previous attempt. Removed.